webpack.config.js

```diff
@@ -0,0 +1,24 @@
+// const path = require('path');
+
+module.exports = {
+  watch: true,
+  entry: './src/index.js',
+  output: {
+    filename: 'main.js',
+  },
+  module: {
+    rules: [{
+      test: /\.css$/i,
+      use: [
+        'style-loader',
+        'css-loader',
+      ],
+    }],
+
+  },
+
+  devServer: {
+    static: __dirname,
+    port: 9000,
+  },
+};
```

Comment:
- [ ] Please add HTML plugins so that webpack  could  compile html files too. Please follow the instructions from the [setting up HtmlWebpackPlugin](https://webpack.js.org/guides/output-management/#setting-up-htmlwebpackplugin) guide.

Reply:
Fixed